inference-engine/tests_deprecated/functional/vpu/common/layers/myriad_layers_blob_test.cpp

```diff
@@ -6,6 +6,7 @@
 #include <ngraph_functions/subgraph_builders.hpp>
 #include "myriad_layers_tests.hpp"
 #include "vpu_tests_config.hpp"
+#include <fstream>
```

Comment:
Is it necessary?

Reply:
yes

Reply:
Could you please explain a bit why? I don't see where you added code to this file that uses something from this header. Maybe I missed something.

Reply:
dependencies between header files. Some includes with fstreams are removed (maybe not in this file, but in includes of this file), so we have to add them explicitly.

Reply:
Thanks for clarification!